app/models/article.rb

```diff
@@ -429,7 +429,7 @@ def evaluate_markdown
   def update_main_image_background_hex
```

Comment:
This is also being triggered in an `after_save` callback which means the record will not be in the database yet if it is new. Can you change this so it's fired in an `after_commit` callback? THANK YOU!